docker/python3.Dockerfile

```diff
@@ -1,7 +1,9 @@
 FROM python:3.6-alpine
 
 RUN mkdir /src \
-  && mkdir /tool
+  && mkdir /tool \
+  && apk add musl-dev gcc  \
```

Comment:
Does pylint have native extensions?

Reply:
Are you asking if there are more dependencies that are required here? Pylint does have a concept of an "extension", but I don't see any that require external libraries.

Reply:
If there are no additional dependencies why do you need musl-dev and gcc?